packages/liquidator/src/proxyTransactionWrapper.js

```diff
@@ -33,7 +33,6 @@ class ProxyTransactionWrapper {
     collateralToken,
     account,
     dsProxyManager = undefined,
-    useDsProxyToLiquidate = false,
```

Comment:
this variable was not actually used. the `useDsProxyToLiquidate` is fed in via the `proxyTransactionWrapperConfig`

Reply:
ahh good catch